app/src/main/java/com/example/android/uamp/viewmodels/NowPlayingFragmentViewModel.kt

```diff
@@ -44,18 +44,23 @@ class NowPlayingFragmentViewModel(
     musicServiceConnection: MusicServiceConnection
 ) : AndroidViewModel(app) {
 
+    // Current media item being played
     val mediaItem = MutableLiveData<MediaItem>().apply {
         postValue(MediaItem.EMPTY)
     }
+    // Current position of the media item being played
     val mediaPosition = MutableLiveData<Long>().apply {
         postValue(0L)
     }
+    // Duration of the media item being played
     val mediaDuration = MutableLiveData<Long>().apply {
         postValue(0L)
     }
+    // Painter resource for the play/pause button in the playback bar
     val mediaButtonRes = MutableLiveData<Int>().apply {
         postValue(R.drawable.ic_album_black_24dp)
     }
+    val isPlaying = MutableLiveData<Boolean>(false)
```

Comment:
nit: remove, I don't think this gets used anywhere

Reply:
`
        isPlaying.postValue(playbackState.isPlaying)
`
This actually get's used in updateState although I doubt this line would be necessary.